actions/tag/calculate-semver/entrypoint/main.go

```diff
@@ -236,3 +236,16 @@ func calculateNextSemver(previous semver.Version, largestChange int) semver.Vers
 	}
 	return semver.Version{}
 }
+
+func setOutput(tag string) {
```

Comment:
Can we rename this function to more clearly capture that it is only writing tags? Initially I thought it was writing any key/value pair, but now I see it is only writing tags.

Maybe call it: `writeTagOutput`?